engines/sci/engine/vm_types.h

```diff
@@ -41,6 +41,9 @@ struct reg_t {
 	SegmentId _segment;
 	uint16 _offset;
 
+	reg_t() = default;
```

Comment:
Why do we need this?

Reply:
Because there still are default constructed objects.